src/main.rs

```diff
@@ -95,13 +97,19 @@ fn _main() -> anyhow::Result<()> {
     }
 
     let opt = Options::from_args_and_env()?;
+    let cfg = config::get_config();
 
     let mut builder = env_logger::Builder::from_env(
         env_logger::Env::default().default_filter_or("warn")
     );
     log_levels::init(&mut builder, &opt);
     builder.init();
 
+    let cfg = cfg.unwrap_or_else(|e| {
+        log::warn!("Config error: {:#}", e);
```

Comment:
Can you rephrase it to `Error in config file '/path/to/config/cli.toml': error text here`?

Reply:
Config file path is added here: https://github.com/edgedb/edgedb-cli/pull/481/files/9a3f395c2c0958204d24f064676302fe24e3a4ea#diff-cba64c21ab992eaad29fce147a08f4560a4769bc14682b8a96081a5fd02dbecdR46
(`#[context(..)]` decorator around `read_config`)